lib/OpenCloud/ObjectStore/Resource/DataObject.php

```diff
@@ -354,6 +382,26 @@ public function delete($params = array())
     {
         return $this->getService()->getClient()->delete($this->getUrl())->send();
     }
+    
+    /**
+     * @param string $source Path (`container/object') of other object to symlink this object to
+     * @return \Guzzle\Http\Message\Response
+     */
+    public function symlink($source)
```

Comment:
Would you mind changing this method name to `makeSymlink` or `createSymlink`? We try to follow verb-noun method naming whenever possible.